SUB-PROPERTY PERMUTATIONS:.md

```diff
@@ -0,0 +1,243 @@
+SUB-PROPERTY PERMUTATIONS:
+
+Axiom: for [B] object_subProperty of [A], any {x,y} pair of instances in B will occur in [A].
+However, it is not the case that any {x,y} in A will occur in [B].
+So, the issue is to find where the logical conditions of Sub_Property [B] violate those of their parent object_property [A]. 
+
+For any Object Property [A] as (Functional, Inverse Functional, Transitive, Symmetric, Asymmetric, Reflexive, Irreflexive),
+Consider whether its sub_Property [B] violates what is maintained by the logical committments of [A].
+Cases are considered in piecemeal fashion examining all possible combinations of some [A] paired with each possible [B].
+Permutation involving inverses of [A] & [B] (A', B' respectively) are forthcoming. 
+
+1. [A] as Functional
+        
+        [A] + [B] as Functional 
+        The functional properties of [B] do NOT violate the functional properties of [A].
+
+        [A] + [B] as Inverse Functional 
+        The inverse functional properties of [B] do NOT violate the functional properties of [A].
+
+        [A] + [B] as Reflexive
+        The reflexive properties of [B] do NOT violate the functional properties of [A].
+
+        [A] + [B] as Irreflexive
+        The irreflexive properties of [B] do NOT violate the functional propertis of [A].
+
+        [A] + [B] as Symmeytric
+        The symmetric properties of [B] do NOT violate the functional properties of [A].
+
+        [A] + [B] as Asymmetric 
+        The asymmetric properties of [B] do NOT violate the functional properties of [A].
+
+        [A] + [B] as Transitive
+        The transitive properties of [B] DO violate the functional properties of [A].
+            PROOF: 
+                -Functionality holds that for object property <R>, {x} can be related to more than one distinct individual {y}.
+                -Transitivity holds that if xRy and yRz then xRz. 
+                -It follows that the functional cardinality of [A] is violated by transitivity in [B].
```

Comment:
Here I just tried to reformulate: this leads to a potential violation since x could relate to multiple y values via chaining, violating the functional constraint.